build-tools/create-packs/Microsoft.Android.Ref.proj

```diff
@@ -11,6 +11,7 @@ by projects that use the Microsoft.Android framework in .NET 5.
 
   <PropertyGroup>
     <PackageId>Microsoft.Android.Ref</PackageId>
+    <PackageType>DotnetPlatform</PackageType>
```

Comment:
Should we put this in https://github.com/xamarin/xamarin-android/blob/master/build-tools/create-packs/Directory.Build.props#L6?